content/blog/2022/2022-02-14-spring-shell.md

```diff
@@ -0,0 +1,295 @@
+## Spring Shell
```

Comment:
Please copy the header of another blog post (called "front matter") and modify it for this blog post. Also, add yourself to the file `data/authors.yml`, please (ideally with a profile picture).